Gemfile.lock

```diff
@@ -18,14 +18,14 @@ GEM
   remote: https://rubygems.org/
   specs:
     aws-eventstream (1.0.3)
-    aws-partitions (1.238.0)
+    aws-partitions (1.233.0)
```

Comment:
Strange that we're downgrading these packages.  Any idea why?

Reply:
No idea, I thought it was strange as well

Reply:
I ran bundle update to get later versions of the gems